parameter_optimization/randomforest.py

```diff
@@ -0,0 +1,40 @@
+import sklearn
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import RandomizedSearchCV, GridSearchCV
+
+
+class LoadData(object):
```

Comment:
I approved the pull request for parameter selection which featured this same class as a stand alone file. Why not just import that class instead of redefining it t avoid duplication?

Also creating that class was ingenious. I love it.

Reply:
Oh and the files paths for this class are not valid. They are written to be accessed from one directory back. Other wise this ran fine